### DISCUSSION TITLE
How save linux settings?

### DISCUSSION THREAD

**Discussion by mynlive** [minios-linux/minios-live#4] [Q&A] [answered]
How save linux settings? if i restart settings .... RIP

**Comment by crim50n** [comment, marked as answer]
There are several options for saving changes:
1. Using FAT32/Ext4 as the file system, permanent mode (default in the boot menu) and the official installation method: https://minios.dev/docs/docs/installation/installation-on-usb.
2. If for some reason the minios/changes folder is not writable, you can use the savechanges command (same as in Slax). For example:
```
sudo savechanges changes.sb
sudo minios-geniso minios.iso changes.sb
```
One command will save the changes in the module, the second will generate an iso with that module.
